.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
.vscode should be ignored, please add it to `.gitingore` then remove this file.

Reply:
done, thank you